packages/core/content-manager/server/src/history/content-types/history-version.ts

```diff
@@ -0,0 +1,66 @@
+import type { Plugin } from '@strapi/types';
+
+const historyVersion: Plugin.LoadedPlugin['contentTypes'][string] = {
+  schema: {
+    uid: 'plugin::content-manager.history-version',
+    modelName: 'history-version',
+    globalId: 'ContentManagerHistoryVersion',
+    kind: 'collectionType',
+    modelType: 'contentType',
+    collectionName: 'strapi_history_versions',
+    info: {
+      singularName: 'history-version',
+      pluralName: 'history-versions',
+      displayName: 'History Version',
+    },
+    options: {
+      reviewWorkflows: false,
+    },
```

Comment:
minor detail, if content type is not visible on the content manager , review workflows will not be enabled on it!